src/tools/fs_utils.rs

```diff
@@ -108,7 +109,8 @@ fn main() {
     let utilcage = Cage{cageid: 0,
                         cwd: interface::RustLock::new(interface::RustRfc::new(interface::RustPathBuf::from("/"))),
                         parent: 0, 
-                        filedescriptortable: interface::RustLock::new(interface::RustHashMap::new())};
+                        filedescriptortable: interface::RustLock::new(interface::RustHashMap::new()),
+                        getgid: AtomicI32::new(-1), getuid: AtomicI32::new(-1), getegid: AtomicI32::new(-1), geteuid: AtomicI32::new(-1)};
```

Comment:
You probably should use the interface declarations here for consistency.

Reply:
Yeah not a huge deal within fs_utils but probably would be good since it's in the interface already.

Reply:
I did not do this because it was giving import errors on VSCode.
But it seems that it just does compile without any issues. 
Fixed.